Notes:
Simulation with 1000 agents takes appx. 40 seconds.

To Do:
Transfer motivation for parameter values in baseline specification from notebook model spec to this notebook.

In [1]:
import pandas as pd

In [4]:
data_frame_test_sim_1000 = pd.read_csv('toy_model_sim_test_1000.csv', sep = '\t')

In [5]:
data_frame_test_sim_1000

Identifier  Period  Identifier.1  Period.1  Years of Education  Choice  \
0              0       0             0         0                  12       2   
1              0       1             0         1                  12       1   
2              0       2             0         2                  12       2   
3              0       3             0         3                  12       1   
4              0       4             0         4                  12       1   
5              0       5             0         5                  12       2   
6              0       6             0         6                  12       1   
7              0       7             0         7                  12       2   
8              0       8             0         8                  12       2   
9              0       9             0         9                  12       2   
10             1       0             1         0                  14       1   
11             1       1             1         1                  14       2   
12             1       2             1         2                  14       2   
13             1       3             1         3                  14       2   
14             1       4             1         4                  14       2   
15             1       5             1         5                  14       1   
16             1       6             1         6                  14       2   
17             1       7             1         7                  14       2   
18             1       8             1         8                  14       1   
19             1       9             1         9                  14       2   
20             2       0             2         0                  12       0   
21             2       1             2         1                  12       0   
22             2       2             2         2                  12       2   
23             2       3             2         3                  12       2   
24             2       4             2         4                  12       2   
25             2       5             2         5                  12       2   
26             2       6             2         6                  12       2   
27             2       7             2         7                  12       1   
28             2       8             2         8                  12       2   
29             2       9             2         9                  12       2   
...          ...     ...           ...       ...                 ...     ...   
9970         997       0           997         0                  12       2   
9971         997       1           997         1                  12       2   
9972         997       2           997         2                  12       2   
9973         997       3           997         3                  12       1   
9974         997       4           997         4                  12       1   
9975         997       5           997         5                  12       2   
9976         997       6           997         6                  12       2   
9977         997       7           997         7                  12       2   
9978         997       8           997         8                  12       2   
9979         997       9           997         9                  12       1   
9980         998       0           998         0                  12       1   
9981         998       1           998         1                  12       2   
9982         998       2           998         2                  12       2   
9983         998       3           998         3                  12       1   
9984         998       4           998         4                  12       2   
9985         998       5           998         5                  12       1   
9986         998       6           998         6                  12       2   
9987         998       7           998         7                  12       2   
9988  

Parameters to keep as in baseline specification:
- part-time experience accumulation g_s1,2,3
- experience depreciation delta

Parameters to change and expected changes:

Parameters in the systematic wage govern the choice between employment (either part-time, or full-time) and nonemployment. They do not determine the choice between part-time and full-time employment since the systematic wage is equal for both options.
- constnat in wage process gamma_0: lower/higher value of the coefficient implies that other components such as accumulated work experience and the productivity shock are relatively more/less important in determining the choice between employment. Increasing the constant for individuals of a certain education level, e.g., high, should result in these indiciduals working more often.
- return to experience gamma_1: lower value of the coefficient implies that accumulated work experience is less relevant in determining the wage in comparison to other factors such as the constant or the productivity shock. Higher coefficients should lead to agents persistently choosing to employment versus non-employment.

The productivity shock:
- productivity shock variances - the higher the variances, the more switching between occupational alternatives should be expected.

Risk aversion:
- risk aversion parameter mu (??)

The labor disutility parameters directly influence:
- benefits - for higher benefits individuals of all education levels would choose non-employment more often
- labor disutility for part-time theta_p - for a higher coefficient, individuals of all education levels would choose to work part-time more often
- labor disutility for full-time theta_f - for a higher coefficient, individuals of all education levels would choose to work part-time more often

Descriptive statistics to look at:
- average part-time, full-time and nonemployment rate - ideally close to population rates, 60%, 27%, 3%
- frequency of each choice per period - ideally more often part-time in early periods, more full-time in later periods
- frequency of each choice over all periods for individuals with different levels of education - ideally, lower educated more often unemployed and in part-time jobs
- average period wages over all individuals - series for all periods
- average period wages series by education